.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
 ## What? and Why?
-> What does this pull request change/fix? Why was it nessessary?
+> What does this pull request change/fix? Why was it necessary?
```

Comment:
In the majority I have fixed it as I go along as I have put PRs in on most repo's